JSDemos/Demos/Charts/AxisLabelsTemplates/description.md

```diff
@@ -0,0 +1,5 @@
+Axis labels are visual elements that display values of the [major axis ticks](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAxisSettings/tick/).
+
+To configure labels for individual axes, specify label settings in the **valueAxis**.[label](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/label/) and **argumentAxis**.[label](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/argumentAxis/label/) objects. To configure labels for all axes, use the **commonAxisSettings**.[label](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAxisSettings/label/) object. Individual settings take precedence over these settings.
+
+This demo shows how to implement labels with custom content. To do this, declare the SVG markup in the [template](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAxisSettings/label/#template) option. You can access a label's raw and formatted values in template data. 
```

Comment:
We're not really implementing new labels, we are just showing something different in them. I thought it would be more straightforward to say it like this: 

```suggestion
This demo shows how to display custom content in axis labels. To do this, declare SVG markup in the [template](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/commonAxisSettings/label/#template) option. You can access a label's original and formatted values from template code. 
```